### DISCUSSION TITLE
为什么不把当月赞助者放在服务器

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3359] [讨论]
放在服务器可以更快更新，免得一些赞助者~~死等烂等~~等不到自己名字出现在PCL2上
可以每次启动启动器后对比缓存sha1，不同就更新缓存

**Comment by iPodtouch6557** [comment]
如果你指的是自建服务器，那么，猫曰：不会制作云服务器相关功能
理由是没有能力抵挡攻击，而且和TZG有关
edit:见 https://github.com/Hex-Dragon/PCL2/discussions/2132#discussioncomment-6313203

**Reply by kong-que-dnf123** [reply]
话说怎么引用可以像你这样引用某issue或discussion的 **某条具体内容** 啊
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/f3e1cfbb-d0d9-4112-bc7a-e3453455c28a)

**Reply by hejiehao** [reply]
其实可以直接把链接粘贴上去（）

**Reply by allMagicNB** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/116283787/98a9e076-f4c3-4d3c-82d6-1b829cd71c15)

**Reply by hejiehao** [reply]
甚至还能跨仓库的

**Comment by mc-124** [comment]
> 如果你指的是自建服务器，那么，猫曰：不会制作云服务器相关功能 理由是没有能力抵挡攻击，而且和TZG有关 edit:见 [#2132 (reply in thread)](https://github.com/Hex-Dragon/PCL2/discussions/2132#discussioncomment-6313203)

可以使用[ksust](https://my.ksust.com/)代替服务器，被攻击了也没事，就是个文件直链平台

**Reply by MoYuan-CN** [reply]
不会用第三方服务的

**Comment by mc-124** [comment]
gitee pages应该也行
